### PR TITLE
Bug 1390112 - Option for filing security bugs

### PR DESCRIPTION
**Pull request by SebastinSanty**

### CODE REVIEW

File: template/en/default/bug/new_bug.html.tmpl [hunk 1]
```diff
@@ -54,6 +54,8 @@
           %]
 
           [% INCLUDE bug_modal/common_new_comment.html.tmpl disable_cols=1 %]
+          <input name="groups" value="core-security" id="group_core-security" type="checkbox"> Many users could be harmed by this security problem:
```
Comment (dylanwh) [inline template/en/default/bug/new_bug.html.tmpl]: Not sure if this is correct in practice, I'll dig the details tomorrow.

Comment (SebastinSanty) [inline template/en/default/bug/new_bug.html.tmpl]: @dylanwh Any updates on this?

**Comment by dylanwh** [comment]
I will take over this one

**Comment by SebastinSanty** [comment]
@dylanwh hmm, great!

**Pull request merged**